build/common.nix

```diff
@@ -17,61 +14,71 @@ with lib;
 
   nixpkgs.config.allowUnfree = true;
 
-  services.openssh.authorizedKeysFiles = mkForce [ "/etc/ssh/authorized_keys.d/%u" ];
-
-  boot.kernelModules = [ "coretemp" ];
-
-  # Prevent "out of sync" errors on the KVM switch.
-  boot.vesa = false;
-  boot.blacklistedKernelModules = [
-    "radeonfb"
-    "radeon"
-    "i915"
-  ];
-  boot.kernelParams = [ "nomodeset" ];
-
   hardware.enableAllFirmware = true;
   hardware.cpu.amd.updateMicrocode = true;
   hardware.cpu.intel.updateMicrocode = true;
 
-  environment.systemPackages = [
-    pkgs.emacs
-    pkgs.sysstat
-    pkgs.hdparm
-    pkgs.sdparm # pkgs.lsiutil
-    pkgs.htop
-    pkgs.sqlite
-    pkgs.iotop
-    pkgs.lm_sensors
-    pkgs.hwloc
-    pkgs.lsof
-    pkgs.numactl
-    pkgs.gcc
-    pkgs.smartmontools
-    pkgs.tcpdump
-    pkgs.gdb
-    pkgs.elfutils
-  ];
+  boot.kernel.sysctl = {
+    # reboot on kernel panic
+    "kernel.panic" = 60;
+    "kernel.panic_on_oops" = 1;
```

Comment:
In my experience this is not advisable at all and causes random reboots once every day or two.

Reply:
I have just refactored this, the infra has been running with this for years. And personally I cannot confirm such a behavior either.